ManualText.txt

```diff
@@ -553,11 +553,70 @@ o	You should arrange desk coverage to complete such calls.
+•	Motor vehicle stops: Log location, vehicle involved, infraction, any other pertinent info given by officer.
+•	Vehicle tows (Report Exec and separate tow log): Log vehicle involved, towed from and to.
+•	Escorts: Log time of original call, name of caller, from and to, arrival time, clear time.
+•	Trouble alarms: Log location of alarm, info from fire computer screen, the action taken.
+•	Any phone calls that the officer asks you to make (for example a call to a student to move their car from a fire lane): 	Add a brief description in the notes of what was said, (i.e. left message, indicated car would be ticketed/towed if not 	moved in next 15 min).
+•	Any complaint or report someone makes of an observation that pertains to the department: Log details and notify 	officer.
+•	All admittance requests: Log name of person, area of admittance, whether access was granted or denied.
+•	Anything else the officer asks you to log.
+
+5.2	MasterFiles (General Use Computer #2)
+MasterFiles is the main software used on the #2 computer. MasterFiles contains most of the information that you will need while on duty, such as cell phone numbers, dorm rooom addresses, vehicles listings, bike listings, etc. It is also how we print short term vehicles passes. You can log into MasterFiles using your standard Gordon username and password. You should log into MasterFiles at the beginning of your shifts to have the the information readily available. 
+
+NOTE: All of the information in MasterFiles is considered confidential and should not be shared with anyone outside of the office. 
+
+5.3	Email, Go! site, Operator Look-up Tool, and Blackboard
+At the beginning of your shift you will also log into your Gordon email, the Go! site and Blackboard. These should all be opened on the general use computer. The #1 computer is for Rerpot Exec ONLY. If you open Google Chrome the tabs for these sites will open automatically. Google Chrome will also have all of the bookmark links that you might use, so it is always best to use this internet browser. 
+
+5.4	Other use
+You can also use the General Use Computer for other work related things, such as creating a word document or looking up information on the internet. You should not use either computers for personal use and you should NEVER adjust the volumes on either of the dispatch computers. It is allowed to use a computer and/or browse the internet during your shift, but you must bring in a personal laptop to do this. 
+
+5.5	Back-up logs
+In the case that Report Exec is not working or the computers are down, you will continue to log everything you would in report exec on the backup paper logs. These can be found in the black tray above the Keurig. When you are done with your shift there is a return location for them just under the blank logs. When Report Exec is again available for use, the paper logs must be hand entered into the system for our records. 
```

Comment:
You can plan on Report Exec not working when the network is down for scheduled network upgrades.